File: src/controllers/orders.controller.js
const ordersService = require('../services/orders.service');

function create(req, res, next) {
  try {
    const userId = req.user.id; // берём из JWT токена
    const { items } = req.body;
    const order = ordersService.create(userId, items);
    res.status(201).json(order);
  } catch (err) {
    next(err);
  }
}

function getMyOrders(req, res, next) {
  try {
    const userId = req.user.id;
    const orders = ordersService.getByUser(userId);
    res.status(200).json(orders);
  } catch (err) {
    next(err);
  }
}

module.exports = { create, getMyOrders };

File: app/controllers/orders_controller.py
from typing import List
from app.models.schemas import OrderCreate, OrderResponse
from app.services import orders_service


def create(user_id: int, data: OrderCreate) -> OrderResponse:
    order = orders_service.create_order(user_id, data)
    return OrderResponse(**order)


def get_my_orders(user_id: int) -> List[OrderResponse]:
    orders = orders_service.get_user_orders(user_id)
    return [OrderResponse(**o) for o in orders]
